### DISCUSSION TITLE
Storage suspended

### DISCUSSION THREAD

**Discussion by molotbar** [vercel/community#4215] [Help]
### Summary

My team was on a Pro plan trial.
Today the trial ended so our Postgres DB got suspended with the following error:
> This database has been suspended.
> This team does not have a valid payment method.
> [Update your payment details](https://vercel.com/teams/five-dragons/settings/billing) to resolve this issue or contact support if you are experiencing issues paying.
> If no action is taken within 30 days, this database and all data will be permanently deleted.

I then added a payment method in the team settings
But the problem is that the error is still there and the storage is still suspended

Has anyone encountered this problem before?

### Steps to Reproduce

- Pro trial ended
- Storage suspended with payment method missing
- Added payment method
- Storage still suspended

**Comment by amyegan** [comment]
Sorry you're having trouble with your account. I found one team associated with your GitHub username. It is currently downgraded because of an expired Pro Trial. You should see a banner near the top of that page when you go to the team's dashboard.

<img width="579" alt="Banner saying 'Your pro trial has expired. Would you like to upgrade to pro?'" src="https://github.com/vercel/community/assets/8247105/da51a09f-147d-4bd0-b3b8-bf0a8c04ff6b">


You can click the "Upgrade Now" button in the banner, or click the "Upgrade" button in the Plan section of your team's Billing settings page. Once you complete the upgrade to a full Pro account, you will have access to all Pro features.
